cmd/integration_test.go

```diff
@@ -461,6 +461,25 @@ func getCloudTestEndChecker(t *testing.T, expRunStatus lib.RunStatus, expResultS
 	return srv
 }
 
+func getSimpleCloudOutputTestState(
+	t *testing.T, script []byte, cliFlags []string,
+	expRunStatus lib.RunStatus, expResultStatus cloudapi.ResultStatus, expExitCode int,
+) *globalTestState {
+	srv := getCloudTestEndChecker(t, expRunStatus, expResultStatus)
```

Comment:
Maybe also marks this as a helper, WDYT? 

```suggestion
	t.Helper()
	
	srv := getCloudTestEndChecker(t, expRunStatus, expResultStatus)
```

Reply:
I find `t.Helper()` to be unhelpful when debugging test errors, it just hides error information that makes it harder to diagnose the root cause. Especially when "helpers" themselves make assertions and can fail the test, it is generally more harmful than helpful to use it.